package.json

```diff
@@ -11,11 +11,13 @@
     "@types/react": "^18.0.0",
     "@types/react-dom": "^18.0.0",
     "gapi-script": "^1.2.0",
+    "moment": "^2.29.4",
     "node-sass": "^7.0.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-google-login": "^5.2.2",
     "react-icons": "^4.4.0",
+    "react-moment": "^1.1.2",
```

Comment:
no need of this library, ``moment`` is enough

Reply:
react-moment - removed

Reply:
Sorry, my mistake, forgot to update package file